src/main/java/io/reactivex/Flowable.java

```diff
@@ -15774,34 +15761,31 @@ public final Completable switchMapCompletableDelayError(@NonNull Function<? supe
     }
 
     /**
-     * Returns a Flowable that only emits those items emitted by the source Publisher that are not followed
-     * by another emitted item within a specified time window.
+     * Alias to {@link #debounce(long, TimeUnit)}
      * <p>
-     * <em>Note:</em> If the source Publisher keeps emitting items more frequently than the length of the time
-     * window then no items will be emitted by the resulting Publisher.
+     * Returns a Flowable that mirrors the source Publisher, except that it drops items emitted by the
+     * source Publisher that are followed by newer items before a timeout value expires. The timer resets on
+     * each emission.
      * <p>
-     * <img width="640" height="305" src="https://raw.github.com/wiki/ReactiveX/RxJava/images/rx-operators/throttleWithTimeout.png" alt="">
```

Comment:
Please keep the marbles as they were because they contain the operator name.

Reply:
Okay, done. However, I don't understand why there should be two copies of the same image with different names. Take a look at the javadoc for `share()` operator: it is an alias for `publish().refCount()` and it has a link to https://raw.github.com/wiki/ReactiveX/RxJava/images/rx-operators/publishRefCount.png.

Reply:
There are a lot of copy-paste and outdated diagrams to fix because of wrong operator names. At least these were named according to their operator. Leaving it as is the simpler task as the alternative, combining the two image by listning both names would be a different undertaking.